css/main.css

```diff
@@ -0,0 +1,59 @@
+body{
+    margin : 0;
+}
+.navigation ul li{
+    float: left;
+    list-style: none;
+    color: white;
+    padding:10px;
```

Comment:
no need for two padding attributes. if you want `padding: 10px;` and `padding-top: 3px;` simply write it as `padding: 3px 10px 10px;` this means 3 on top, 10 on sides , and 10 on bottom.